co-authors-plus.php

```diff
@@ -740,10 +740,10 @@ function posts_where_filter( $where, $query ) {
 
 					$maybe_both_query = $maybe_both ? '$0 OR' : '';
 
-					$where = preg_replace( '/\s\b(?:' . $wpdb->posts . '\.)?post_author\s*IN\s*\(' . $id . '\)/', ' (' . $maybe_both_query . ' ' . $terms_implode . ')', $where, -1 ); // ' . $wpdb->postmeta . '.meta_id IS NOT NULL AND
+					$where = preg_replace( '/\s\b(?:' . $wpdb->posts . '\.)?post_author\s*IN\s*\(' . $id . '\)/', ' (' . $maybe_both_query . ' ' . $terms_implode . ')', $where, 1 ); // ' . $wpdb->postmeta . '.meta_id IS NOT NULL AND
```

Comment:
Were you able to trigger in your testing this particular section of the code? Curious because it slightly differs from the other `$where` statements on the pattern matching.

Reply:
Yes, you can trigger this by logging into a Contributor account and looking at `/wp-admin/edit.php?author_name=admin` however with additional testing, this change does not fix or improve anything, so I'm going to revert this proposed changed just for this line.